tests/commands/test_commit_command.py

```diff
@@ -324,6 +324,68 @@ def test_commit_when_nothing_to_commit(config, mocker: MockFixture):
     assert "No files added to staging!" in str(excinfo.value)
 
 
+@pytest.mark.usefixtures("staging_is_clean")
+def test_commit_when_nothing_added_to_commit(config, mocker: MockFixture):
+    prompt_mock = mocker.patch("questionary.prompt")
+    prompt_mock.return_value = {
+        "prefix": "feat",
+        "subject": "user created",
+        "scope": "",
+        "is_breaking_change": False,
+        "body": "",
+        "footer": "",
+    }
+
+    commit_mock = mocker.patch("commitizen.git.commit")
+    commit_mock.return_value = cmd.Command(
+        'nothing added to commit but untracked files present (use "git add" to track)',
```

Comment:
we might encounter issue if the language is not set to en_US. probably worth setting it up for this test

Reply:
Got implemented since last year by newer commit 471b778e18937cb0291bf266fa1cd2a58a53e081, but locale not taken into account